### DISCUSSION TITLE
How to disable dimness of files that listed in "always_show_by_pattern" config option?

### DISCUSSION THREAD

**Discussion by nowaylifer** [nvim-neo-tree/neo-tree.nvim#1550] [Q&A]
I have
```lua
filtered_items = {
  hide_dotfiles = true,
  always_show_by_pattern = {
    ".env*"
  },
}
```
files are shown, but they're dim though

![image](https://github.com/user-attachments/assets/aec73f1e-22c7-4137-a629-475c236cf647)

**Comment by ChrisTaylorDeveloper** [comment]
I would be interested in a solution to this also.  It would be nice if 'always shown' files were coloured.
